tests/setup-snapshot.yml

```diff
@@ -0,0 +1,12 @@
+- hosts: all
```

Comment:
What is the purpose of this file? It does not seem to be used in this PR.

Reply:
> What is the purpose of this file? It does not seem to be used in this PR.

It is used by CI for test performance purposes.  The first time CI runs a test for the role for a particular platform, it will create a snapshot image which will be pre-populated with the latest repo metadata and packages, and will have the epel repo installed (but disabled - the role tests will need to enable epel when needed).  If the role provides the file tests/setup-snapshot.yml, then this will also be run to prepare the snapshot.  One of the primary things this file does is to install packages which will be required by CI, because package installation is usually one of the most time consuming tasks.  And because we don't want to duplicate the list of packages in this file *and* in the role somewhere, the role has been somewhat refactored so that we can use the list of packages defined by the role in setup-snapshot.yml by moving the code that establishes the list of packages to `tasks/set_facts.yml`, then calling this file from setup-snapshot.yml by using include_role with `tasks_from: set_facts.yml`.  CI will run the test by adding the arguments `--use-snapshot` and `--setup-yml tests/setup-snapshot.yml`

https://github.com/linux-system-roles/tox-lsr#qemu-testing